service/src/main/java/bio/terra/user/app/controller/PublicApiController.java

```diff
@@ -0,0 +1,52 @@
+package bio.terra.user.app.controller;
+
+import bio.terra.user.api.PublicApi;
+import bio.terra.user.app.configuration.VersionConfiguration;
+import bio.terra.user.model.ApiVersionProperties;
+import bio.terra.user.service.status.StatusService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class PublicApiController implements PublicApi {
+  private final StatusService statusService;
+  private final VersionConfiguration versionConfiguration;
+
+  @Autowired
+  public PublicApiController(
+      StatusService statusService, VersionConfiguration versionConfiguration) {
+    this.statusService = statusService;
+    this.versionConfiguration = versionConfiguration;
+  }
+
+  @Override
+  public ResponseEntity<Void> serviceStatus() {
+    return new ResponseEntity<>(
+        statusService.getCurrentStatus() ? HttpStatus.OK : HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
can we just return OK in this PR for simplicity?

TPS is also just returning OK https://github.com/DataBiosphere/terra-policy-service/blob/main/service/src/main/java/bio/terra/policy/controller/PublicApiController.java.